client/post-editor/media-modal/edit/_style.scss

```diff
@@ -0,0 +1,47 @@
+.editor-media-modal-edit__content {
+	background: black;
```

Comment:
It could be darkened a bit more with `darken()` if `$gray-dark`is not enough. (An image editor naturally prefers a darker, more neutral tone.)